packages/mdxast/test/test.js

```diff
@@ -30,7 +33,11 @@ const parseFixture = str => {
 }
 
 test('it parses a file', () => {
-  const result = parseFixture(fixture)
+  const result = parseFixture(fixture.basic)
+  expect(result).toMatchSnapshot()
```

Comment:
Could you use babel to parse the result? It's much more reliable than having a snapshot test that will be guaranteed to fail on every little change.

Reply:
good, I can do that :v:

Reply:
👌